x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -110,6 +110,9 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
     apiKeyOwner: {
       type: 'keyword',
     },
+    apiKeyCreatedByUser: {
+      type: 'boolean',
+    },
```

Comment:
Do we need to search/aggregate by this field? If so, LGTM. Otherwise, I'd avoid adding it to the mapping.

Reply:
I don't think we need to search on it rn, I will remove the mapping. Thanks!